things/service.go

```diff
@@ -125,7 +119,7 @@ func (svc service) CreateThings(ctx context.Context, token string, cls ...mgclie
 		})
 	}
 	if _, err := svc.auth.AddPolicies(ctx, &policies); err != nil {
-		return nil, errors.Wrap(errAddPolicies, err)
+		return nil, errors.Wrap(svcerr.ErrCreateEntity, err)
```

Comment:
Also add `svcerr.ErrAddPolicies`

Reply:
We do not want ```svcerr.ErrAddPolicies``` to be sent to a user, that is why we send ```svcerr.ErrCreateEntity``` which is what the user will see. errAddPolicies is left as an internal error which will be logged but we do not want it to be sent along with the response